src/endpoints/file/copyFile.ts

```diff
@@ -0,0 +1,79 @@
+import Elysia from 'elysia';
+import { isPathValid } from '../../utils/pathUtils';
+import { getConfig } from '../../utils/config';
+import { join } from 'path';
+import { getFile, writeFile } from '../../utils/fileUtils';
+import { fileAlreadyExistsMsg } from '../../constants/commonResponses'
+
+
+export function addCopyFileEndpoint(app: Elysia) {
+  return app.post('file/copy', async ({ query, set }) => {
+
+    // Verify that the SOURCE file path is valid
+    let relativeSourcePath = query.source ? (query.source as string) : null;
+
+    // Get the DESTINATION file path
+    let relativeDestinationPath = query.destination ? (query.destination as string) : null;
+
+    let overwrite = false;
+
+    if (query.overwrite) {
+      if (query.overwrite === "true")
+        overwrite = true;
+      else if (query.overwrite === "false")
+        overwrite = false;
+      else {
+        set.status = 400;
+        return 'Overwrite must be one of "true" and "false"';
```

Comment:
```
Parameter "overwrite" must be either "true" or "false"
```

Reply:
Right. Made this message a constant in `commonResponses.ts` and am using that.